lib/util/index.js

```diff
@@ -403,7 +403,7 @@ function buildLogLevels(logLevel) {
   if (typeof logLevel === 'string') {
     return {
       default: logLevel,
-      media: logLevel,
+      media: 'debug', // TODO(mpatwardhan): restore before release.
```

Comment:
Didn't you already do this in [createLocalTracks](https://github.com/twilio/twilio-video.js/pull/1068/files#diff-10abc4ab7f10b4544b8f050f56e9e9c3R16)?

Reply:
I found that change in createLocalTracks wasn't sufficient - it wasn't propagated to `Track`. This ensures that all tracks get it.

Reply:
@makarandp0 In that case you don't need to do it in `createLocalTracks`.